Content.Server/Disposal/Unit/EntitySystems/DisposableSystem.cs

```diff
@@ -115,7 +122,10 @@ public void ExitDisposals(EntityUid uid, DisposalHolderComponent? holder = null,
                 {
                     if (meta.EntityPrototype.Components.ContainsKey("Fragile"))
                     {
-                        Log.Error("SPLAT");
+                        var puddle = String.Format("{0}", meta.EntityPrototype.Components["Fragile"].Mapping["puddle"]);
+                        var solution = new Solution(puddle, 10);
+                        _audio.PlayPvs(new SoundPathSpecifier("/Audio/Effects/Fluids/splat.ogg"), uid, AudioParams.Default.WithVariation(0.2f).WithVolume(-4f));
```

Comment:
use the existing component's sound and dont hardcode params
spill system or something

Reply:
Neither egg, tomato, or watermelon had an existing sound as a component so I added them.